test/utils.hpp

```diff
@@ -276,13 +276,12 @@ std::pair<bool, double> compareEqualLaunchKernel(DDataType*  deviceD,
     auto toDouble
         = [](DDataType const& val) { return static_cast<double>(static_cast<float>(val)); };
 
-    auto eps = toDouble(std::numeric_limits<DDataType>::epsilon());
```

Comment:
epsilone of type is not a good option to measure the difference between data.

epsilone<double> = 2.22045e-14 , which is too small